modell/src/main/kotlin/no/nav/tilbakekreving/tilstand/AvventerFagsysteminfo.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.tilbakekreving.tilstand
+
+import no.nav.tilbakekreving.Tilbakekreving
+
+object AvventerFagsysteminfo : Tilstand {
+    override val navn: String = "AvventerFagsysteminfo"
```

Comment:
Blir denne en konstant? I så fall burde det vare en annen naming?
```suggestion
    override val NAVN: String = "AvventerFagsysteminfo"
```

Reply:
Nja, det er ikke en konstant, men en getter i klassen. navn er mer riktig i en kotlin setting